samples/openshift-cloud-terminal.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: workspace.che.eclipse.org/v1alpha1
+kind: Workspace
+metadata:
+  name: openshift-terminal
```

Comment:
is it "openshift-terminal" or "openshift-cloud-terminal" ??

Reply:
it's just a sample, the actual value will be generated by OpenShift Console. 
But at this point to makes sense to align it with plugin name. I'm going to rename it to openshift-cloud-terminal until another name won't be chosen under thread https://github.com/che-incubator/che-workspace-operator/pull/70#discussion_r424388778